pkg/ride/converters_test.go

```diff
@@ -2031,7 +2031,7 @@ func (a *CreateAliasWithSigTestSuite) SetupTest() {
 
 func (a *CreateAliasWithSigTestSuite) Test_alias() {
 	rs, _ := a.f(proto.MainNetScheme, a.tx)
-	a.Equal(rideString(a.tx.Alias.String()), rs["alias"])
+	a.Equal(rideString(a.tx.Alias.Alias), rs["alias"])
```

Comment:
Is it correct? (The previous result was `alias:W:testalias`)

Reply:
Yes, it is correct. Like in Scala.